packages/website/src/components/hooks/useFocus.ts

```diff
@@ -0,0 +1,14 @@
+import React, { useRef } from 'react';
+
+function useFocus<T extends HTMLInputElement = HTMLInputElement>(): [
```

Comment:
Nit: the `T` is never used.

```suggestion
function useFocus(): [
```